packages/grpc-web/package.json

```diff
@@ -27,10 +27,10 @@
   "devDependencies": {
     "@types/google-protobuf": "~3.7.0",
     "command-exists": "~1.2.8",
-    "google-closure-compiler": "~20200224.0.0",
-    "google-closure-deps": "~20210601.0.0",
-    "google-closure-library": "~20210808.0.0",
-    "google-protobuf": "~3.14.0",
+    "google-closure-compiler": "^20220301.0.0",
+    "google-closure-deps": "^20220202.0.0",
+    "google-closure-library": "^20220202.0.0",
+    "google-protobuf": "~3.19.4",
```

Comment:
I'm curious why some of the JS dependency has to be updated for building on arm64? Are they somehow relevant?

Reply:
Yes, previous versions resulted in build failure on arm64

Reply:
Here is the specific build error if using an older version of the closure compiler dependency under arm64:
```
#0 88.56 /github/grpc-web/packages/grpc-web/node_modules/.bin/google-closure-compiler --js=exports.js --js=../../javascript --js=node_modules/google-closure-library --entry_point=grpc.web.Exports --externs=externs.js --dependency_mode=PRUNE --compilation_level=ADVANCED_OPTIMIZATIONS --generate_exports --export_local_property_definitions --js_output_file=index.js
#0 90.60 Error: EISDIR: illegal operation on a directory, read
#0 90.60     at Object.readSync (fs.js:568:3)
#0 90.60     at tryReadSync (fs.js:353:20)
#0 90.60     at Object.readFileSync (fs.js:390:19)
#0 90.60     at _Sd (/github/grpc-web/packages/grpc-web/node_modules/google-closure-compiler-js/jscomp.js:7552:192)
#0 90.60     at YSd (/github/grpc-web/packages/grpc-web/node_modules/google-closure-compiler-js/jscomp.js:9180:449)
#0 90.60     at sS (/github/grpc-web/packages/grpc-web/node_modules/google-closure-compiler-js/jscomp.js:2442:29)
#0 90.60     at vS (/github/grpc-web/packages/grpc-web/node_modules/google-closure-compiler-js/jscomp.js:3940:44)
#0 90.60     at /github/grpc-web/packages/grpc-web/node_modules/google-closure-compiler-js/jscomp.js:4181:46
#0 90.60     at CompilerJS.run (/github/grpc-web/packages/grpc-web/node_modules/google-closure-compiler/lib/node/closure-compiler-js.js:54:17)
#0 90.60     at /github/grpc-web/packages/grpc-web/node_modules/google-closure-compiler/cli.js:164:31 {
#0 90.60   errno: -21,
#0 90.60   syscall: 'read',
#0 90.60   code: 'EISDIR'
#0 90.60 }
#0 109.6 npm ERR! code ENOENT
#0 109.6 npm ERR! syscall access
#0 109.6 npm ERR! path /github/grpc-web/packages/grpc-web/node_modules/grpc-web.js
#0 109.6 npm ERR! errno -2
#0 109.6 npm ERR! enoent ENOENT: no such file or directory, access '/github/grpc-web/packages/grpc-web/node_modules/grpc-web.js'
#0 109.6 npm ERR! enoent This is related to npm not being able to find a file.
#0 109.6 npm ERR! enoent 
```

Reply:
Aha interesting! Thanks for the info!

What about `google-closure-library`? What that cause an error is left unchanged?

Reply:
Good catch! I tested again and it seems only `google-closure-compiler` needs to be bumped for the build to succeed.